### ISSUE TITLE
[A request to add a new feature] Blocking the addition of the same input again

### ISSUE THREAD

**Issue by marciniuk** [SaDiablo/GenshinCompanion#7]
It would be nice if the program blocked the addition of such input that is already in the database. It is also possible that it crashes when we exceed the 90 wishy number without drawing 5* item, but I haven't tested it because it's too big log cleaning.

**Comment by SaDiablo** [comment]
Allowing items being already there is by design, as if you draw 10pull there can be situation like this:

![image](https://user-images.githubusercontent.com/24566138/113308124-284aa700-9306-11eb-9dbe-dffbc2bec549.png)
And this perfectly valid outcome 

If you need you can manually delete single wish from list by pressing right mouse button and Remove
I was thinking about adding some visual grouping to the 10pulls in the future

Also i couldn't reproduce the problem with exceeding 90 wishes without 5* drop - it's simply showing negative numbers and stops showing 5* Rate which allows people to store above real wish limits (for example to quickly calculate character rates to their total by adding dummy wishes or some other creative ways of using it). 
![image](https://user-images.githubusercontent.com/24566138/113308592-9d1de100-9306-11eb-9c01-d1f88d5e87a2.png)

If you want to test it don't worry about overriding your existing data - saving wishes is manual only :)

**Comment by marciniuk** [comment]
How about "locking" the draw, for example, so that you can add up to 9x in a row of 3-star elements?

**Comment by SaDiablo** [comment]
Good idea, I'm gonna consider adding it as a part of a validity checks in future. For now I'm trying to rebuild this app in AvaloniaUI - if you wanna see progress check Action artefacts, as I'm open for suggestions :D
